packages/editor/src/components/collab-sidebar/style.scss

```diff
@@ -135,6 +135,40 @@
 	display: block;
 }
 
+.editor-collab-sidebar-panel__skip-link {
```

Comment:
Remove any unnecessary styles. The `Button` component should have a variety of styles by default.